gateway/modules/schema/schema.go

```diff
@@ -39,7 +40,14 @@ func (s *Schema) SetConfig(c config.DatabaseSchemas, project string) error {
 	defer s.lock.Unlock()
 	s.dbSchemas = c
 	s.project = project
-
+	s.dbAliasDBTypeMapping = make(map[string]string)
+	for _, databaseSchema := range c {
+		dbType, err := s.crud.GetDBType(databaseSchema.DbAlias)
```

Comment:
Instead of this, why don't you simply make a set db-config method and use that to populate mapping